operations/patch.go

```diff
@@ -292,10 +298,21 @@ func PatchFile() cli.Command {
 		),
 		Before: mergeBeforeFuncs(
 			autoUpdateCLI,
-			requireFileExists(diffPathFlagName),
 			mutuallyExclusiveArgs(false, patchDescriptionFlagName, autoDescriptionFlag),
+			mutuallyExclusiveArgs(false, diffPathFlagName, diffFromPatchFlagName),
+			mutuallyExclusiveArgs(false, baseFlagName, diffFromPatchFlagName),
 		),
 		Action: func(c *cli.Context) error {
+			println("Hi 307")
+
+			diffFromPatch := c.String(diffFromPatchFlagName)
+			if diffFromPatch == "" {
+				path := c.String(diffPathFlagName)
+				if _, err := os.Stat(path); os.IsNotExist(err) {
+					return errors.Errorf("file '%s' does not exist %s ", path, diffFromPatch)
```

Comment:
I don't quite understand what this error is saying. Is it saying that both the requested diff path does not exist and there is no patch to diff from? Can we rephrase this?